src/transformers/models/big_bird/configuration_big_bird.py

```diff
@@ -94,13 +94,13 @@ class BigBirdConfig(PretrainedConfig):
 
     ```
 
-        >>> from transformers import BigBirdModel, BigBirdConfig
+        >>> from transformers import BigBirdConfig, BigBirdModel
 
         >>> # Initializing a BigBird google/bigbird-roberta-base style configuration >>> configuration =
         BigBirdConfig()
 
-        >>> # Initializing a model from the google/bigbird-roberta-base style configuration >>> model =
-        BigBirdModel(configuration)
+        >>> # Initializing a model (with random weights) from the google/bigbird-roberta-base style configuration >>>
```

Comment:
Hi, could you fix a bug. The line `>>> model =` should be in a new line. Same for configuration, and same for `BigBirdPegasusConfig`. Thank you :-)

Reply:
This bug is on us, not from your change :-)